### PR TITLE
Adding FAQ command to the bot

### LINKED ISSUE #102: Commands to send info about FAQs
**Who are you?**
Ryan

**What feature do you want?**
A command or commands to send info with the answer to frequently asked questions. Could be done as a listener, but definitely want the ability to manually send things. Such commands as how to use discord markdown, how to get class roles, link to pgg, etc

**Why do you want this feature?**
Mainly because so few people use markdown formatting when sending code blocks and it annoys me

### PR DESCRIPTION
**Pull request by ReginPotter**
# Description

Added a FAQ command that displays FAQs so Faculty and Dev Team Students don't have to.

## Issues

Closes #102

## Type of change

Select one or more of the following:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- [X] Ran the command
       -faq 
       or
       -FAQ

# Checklist:

- [X] All local commits have been pushed to remote
- [X] All changes on the base branch have been merged into this branch, either by rebase or merge
- [X] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: Cogs/AdminCommands.py [hunk 1]
```diff
@@ -155,3 +155,27 @@ async def stop(self, ctx):
...
+    async def faq(self, ctx):
+        """Responds with frequently asked questions
+        Sends a message to the user that contains frequently asked questions
+        
+
+        Outputs:
+            Message to user frequently asked questions in the discord
+        """
+
+        embed=discord.Embed(
```
Comment (CodingPenguin1) [inline Cogs/AdminCommands.py:170]: pycodestyle(E122)

File: Cogs/AdminCommands.py [hunk 2]
```diff
@@ -155,3 +155,27 @@ async def stop(self, ctx):
...
+            description="Here are some frequently asked questions",
+            color=discord.Color.blue())
+        embed.set_author(name="CSE Dev Team", url="http://www.wright.edu/", icon_url="https://d28htnjz2elwuj.cloudfront.net/wp-content/uploads/2019/04/30105443/Wright-State-University-400x400.jpg")
+
+        embed.add_field(name="How do I add a class?", value="Go to the appropriate 'class selection' channel and click the button of your class", inline=False)
+        embed.add_field(name="Need help with coding?", value="If you are in a class higher than 1181, ask questions in your class chat otherwise go to the appropriate help room", inline=False)
+        embed.add_field(name="What can I send in chats?", value="This server is an official discord server of Wright State therefore please do not send any code that pertains to any project/homework in all channels. Please also refrain from any language you ", inline=False)
+        embed.add_field(name="Why wont my role add?", value="Please make sure you have clicked the button. If you have give discord a moment, sometimes discord is slow.", inline=False)
+        await ctx.send(embed=embed)
```
Comment (CodingPenguin1) [inline Cogs/AdminCommands.py:181]: newline

**Review by CodingPenguin1 — CHANGES_REQUESTED** [review]
Very nice. Would move to student commands and make not admin-only

**Review by CodingPenguin1 — APPROVED** [review]
